mslice/tests/histogram_workspace_test.py

```diff
@@ -65,11 +65,11 @@ def test_add_list(self):
         result = self.workspace + list_to_add
         result = result.get_signal()
 
-        line = np.multiply(np.linspace(0, 9, 10), 10)
-        expected_values = np.empty((10, 10))
-        for i in range(10):
-            expected_values[i] = line
-        np.testing.assert_array_almost_equal(expected_values, result, 8)
+        ws_representative_array = np.linspace(0, 99, 100).reshape(10, 10).T
```

Comment:
See related comment on `pixel_workspace_test,py`. I now directly compare the `workspace + list` operation to the equivalent `numpy array + list`. To me, these should be equal - they previously were not.